client/src/components/pages/legislation/index.js

```diff
@@ -1,16 +1,24 @@
 import React, {Component} from 'react';
+import NavBar from '../../shared/navbar';
+import FooterComponent from '../../shared/footer';
 
 class Legislation extends Component{
+
+    componentDidMount(){
+        document.title = "Legislation | Kapitol"
+    }
+
     render(){
```

Comment:
@megelismi I think here's where it starts to clash, because Karthick's recent PR updated this. Will probably be taken care of in the merge conflict though, right?